components/Button/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableHighlight, View } from 'react-native';
+import { StyleSheet } from 'react-native';
+
+export default class Button extends React.Component {
```

Comment:
Since this component is stateless, can we turn this into a functional component?